### PR TITLE
Change order and wording snapshot actions

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

- Makes the "Wipe & restore" button more clear,
- Changes the order of the buttons

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

Before: 
image

After:
image

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts [hunk 1]
```diff
@@ -192,46 +198,36 @@ class HassioSnapshotDialog extends LitElement {
...
-        ${this._snapshot.type === "full"
           ? html`
-              <mwc-button
-                @click=${this._fullRestoreClicked}
-                slot="secondaryAction"
-              >
-                <ha-svg-icon .path=${mdiHistory} class="icon"></ha-svg-icon>
-                Wipe &amp; restore
-              </mwc-button>
+              <div class="button-row">
```
Comment (bramkragten) [inline hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts:217]:
```suggestion
              <div class="button-row" slot="secondaryAction">
```

File: hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts [hunk 2]
```diff
@@ -192,46 +198,36 @@ class HassioSnapshotDialog extends LitElement {
         ${this._error ? html` <p class="error">Error: ${this._error}</p> ` : ""}
 
         <div>Actions:</div>
+        <div class="button-row">
```
Comment (bramkragten) [inline hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts:201]:
```suggestion
        <div class="button-row" slot="primaryAction">
```

File: hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts [hunk 3]
```diff
@@ -245,6 +241,9 @@ class HassioSnapshotDialog extends LitElement {
           display: block;
           margin: 4px;
         }
+        .button-row mwc-button {
+          padding: 8px;
+        }
```
Comment (bramkragten) [inline hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts:246]:
```suggestion
        .button-row {
            display: flex;
            align-content: space-between;
        }
```

File: hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts (hunk 1 above)
Comment (ludeeus) [reply hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts:217]: Slots have no effect?
<https://github.com/home-assistant/frontend/pull/7677/files#diff-a125a832ada409222c511971db1296a29f700a4d9ae9fb35a403c9dd7d75f751R117>

File: hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts [hunk 4]
```diff
@@ -109,7 +109,13 @@ class HassioSnapshotDialog extends LitElement {
       return html``;
     }
     return html`
-      <ha-dialog open stacked @closing=${this._closeDialog} .heading=${true}>
+      <ha-dialog
+        open
+        stacked
+        @closing=${this._closeDialog}
+        .heading=${true}
+        hideactions
```
Comment (bramkragten) [inline hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts:117]:
```suggestion
```

Comment (bramkragten) [reply hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts:117]: Fixed 😄

File: hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts (hunk 1 above)
Comment (bramkragten) [reply hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts:217]: Why did you choose to hide the actions?

Comment (ludeeus) [reply hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts:217]: Because it's up to 4 actions, so a simple primary/secondary does not make sense?

Comment (bramkragten) [reply hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts:217]: That's what the `stacked` attr on the dialog is for

**Review by bramkragten — APPROVED** [review]

File: hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts [hunk 5]
```diff
@@ -245,6 +240,10 @@ class HassioSnapshotDialog extends LitElement {
           display: block;
           margin: 4px;
         }
+        .button-row {
+            display: flex;
+            align-content: space-between;
```
Comment (bramkragten) [inline hassio/src/dialogs/snapshot/dialog-hassio-snapshot.ts:245]:
```suggestion
            justify-content: space-between;
```

**Review by bramkragten — APPROVED** [review]

**Pull request merged**